### PR TITLE
request Ubuntu image

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: config/prod/iAtlas-large.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "r5.12xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "1000"
+  # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
+  AMIId: "ami-0565af6e282977273"
```
Comment (thomasyu888) [inline config/prod/iAtlas-large.yaml]: Comment out

File: config/prod/iAtlas-large.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+# For CI system (do not change)
+hooks:
+  before_create:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-ubuntu-v1.yaml --create-dirs -o templates/remote/managed-ec2-ubuntu-v1.yaml"
+  before_update:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-ubuntu-v1.yaml --create-dirs -o templates/remote/managed-ec2-ubuntu-v1.yaml"
+  after_create:
+    - !ec2_notify "{{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}"
+
```
Comment (thomasyu888) [inline config/prod/iAtlas-large.yaml]: Remove the extra one.

File: config/prod/iAtlas-large.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  # Default account settings, leave alone unless you plan to override
+  VpcName: {{stack_group_config.default_vpc}}
+  KeyName: {{stack_group_config.default_key_pair}}
+  AMIId: {{stack_group_config.default_ubuntu_ami}}
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "r5.12xlarge"
```
Comment (thomasyu888) [inline config/prod/iAtlas-large.yaml]: Also this instance isn't supported. @zaro0508 can you add support?

Comment (andrewelamb) [reply config/prod/iAtlas-large.yaml]: Is there a list of supported instances?

Comment (thomasyu888) [reply config/prod/iAtlas-large.yaml]: https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-ubuntu-v1.yaml#L15

Comment (andrewelamb) [reply config/prod/iAtlas-large.yaml]: changed to r4.16xlarge

**Pull request merged**